conf/messages

```diff
@@ -130,8 +130,10 @@ group.tax-service.answer.hint=If you select ‘Yes’ we will add new clients as
 group.tax-service.select.error=Select the tax type for this access group
 
 group.tax-service.review.h1=All “{0}” clients have been selected
+group.tax-service.review.h2=You have selected all “{0}” clients
```

Comment:
If this is replacement content, and the message key isn't used anywhere else - you should delete the original content and reuse the message key.

Reply:
Ok I was leaving it in for the title but I've changed it

Reply:
Ahh I see, well the page title normally follows the pattern of [1st Heading of page] - [Service Name] - GOV.UK
https://design.tax.service.gov.uk/hmrc-design-patterns/page-title/

So by changing the H1, the title should change as well. Sorry that won't have been clear to you from the ticket 😅